src/modules/ROOT/pages/changelog/202209/08.json

```diff
@@ -0,0 +1,110 @@
+      ],
+      "why": [
+        "to enable configurators to hide filtering on record lists."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/8ebd25c4b1596af467578848fe9d4eb295d84c80"
+      ]
+    }
+  ],
+  "changed": [
+    {
+      "what": [
+        "the link to the reminder record, it was added in the reminder sidebar list/edit.",
+        "Enabled the ability to un-dismiss reminders.",
+        "Added a date input field to set the date and time of the reminder.",
+        "General UI/UX improvements for reminders."
+      ],
+      "why": [
+        "to make reminders more user friendly and easier to use by having the option to set reminder in more than 24h and to have the option to link a record to a reminder and also to un-dismiss a reminder if needed."
+      ],
+      "how": [
+        "by updating the reminders logic and configuration in our code base."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/abc8ad4a5f5ebff0c9b21d9af9dd25cf1e0f2fba",
+        "https://github.com/cortezaproject/corteza/pull/899/commits/91a86acf75d11626fbec716dde1ba2f2305bbd1c",
+        "https://github.com/cortezaproject/corteza/pull/899/commits/5a559d2be90f54db1806110b0231d27e872b530f",
+        "https://github.com/cortezaproject/corteza/pull/885/commits/6e1dca23d00d9d44ca1c7febd06d49b28de94d99",
+        "https://github.com/cortezaproject/corteza/pull/885/commits/86452eed40176a83bbc24f12ebae66c0f3f3a04e",
+        "https://github.com/cortezaproject/corteza/pull/885/commits/178e64dc1ea190e42eed530eaccd791854c057f1"
+      ]
+    }
+  ],
+  "fixed": [
+    {
+      "what": [
+        "date multi value filtering using contains/not contains."
+      ],
+      "how": [
+        "by adding explicit type casting for date time values to cast into jsonb format while parsing the record filter query."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/903/commits"
+      ]
+    },
+    {
+      "what": [
+        "select field translations not being shown unless the module is saved."
+      ],
+      "how": [
+        "by updating the Select/dropdown configurator with a default initial state of the field options that are already saved."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/1706e5206470f7da9fb61830c730b8aca38d18f5"
+      ]
+    },
+    {
+      "what": [
+        "gauge chart metric functions not showing correct results when report produced multiple data frames."
+      ],
+      "how": [
+        "by setting the dimension field to deletedBy which produces at most one dataset, since the gauge only shows one value."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/883/commits/1866097b4319156d9a3b732ad5d591852868354a"
+      ]
+    },
+    {
+      "what": [
+        "issue of role with invalid ID stored in the local storage from displaying of the permission pages."
```

Comment:
Can you focus on the issue from the user's perspective; why would a user consider this an issue?